source/application/BioXAS/BioXAS.h

```diff
@@ -8,12 +8,16 @@ namespace BioXAS
 {
 	/// Handles the XRF detector choice.  Available choices are None, KETEK, and 13-element Ge.
 	enum FluorescenceDetector {
-
 		NoneXRFDetector = 0,
 		FourElementVortexDetector = 1,
 		Ge13ElementDetector = 2
 	};
 
+	/// Enum to describe each of the three BioXAS beamlines.
+	enum Beamline {
```

Comment:
Okay.  It's not the end of the world or anything, if we don't come up with something better before we merge it in, not a big deal.